### ISSUE TITLE
Kill and restart app during test

### ISSUE THREAD

**Issue by fylyppo** [leancodepl/patrol#1787]
### Use case

I want to check if logged user data is persisted after reopening app.

### Proposal

I can see that looping `patrolTest()` is possible and it works very similar to killing and reopening app during one test case. Maybe that could be some idea for implementation.

**Comment by bartekpacia** [comment]
This sounds similar #1374. Give it a read.

**Comment by bartekpacia** [comment]
> I can see that looping `patrolTest()` is possible and it works very similar to killing and reopening app during one test case. Maybe that could be some idea for implementation.

Yes but no app state is preserved then. And you want to do something in _app run 1_, then kill the app and run it again, and in _app run 2_ you want to perform some assertion knowing that _app run 1_ was executed before.

This sounds like a hard thing to implement well, but I didn't give it much thought.

cc @mateuszwojtczak

**Comment by Kendru98** [comment]
https://github.com/leancodepl/patrol/issues/1374 - Will close this as duplicate, because there is more idea's and comments in this (newer one) issue.
